Remove top warnings and bump version to 1.0.0 (#244)
freedomofpress/securedrop-workstation-docs: docs/general/status.rst

SecureDrop Workstation Project Status
=====================================

SecureDrop Workstation is currently in active development.

> Freedom of the Press Foundation has operated a pilot program with a limited

Reviewer: ```suggestion
Freedom of the Press Foundation had operated a pilot program with a limited
```

number of organizations whose environments were especially well-suited for

> early testing of the Workstation. This pilot program is now closed.

Reviewer: Maybe?

```suggestion
early testing of the Workstation. This pilot program has now ended.
```
